test/JDBC/input/functions/indexproperty.sql

```diff
@@ -0,0 +1,47 @@
+-- should auto-name the primary key index as t1_pkey
+create table t1(c1 int PRIMARY KEY (c1))
```

Comment:
can you also add tests after creating indexes explicitly ?

Reply:
Done.